Documentation/helm-operator.md

```diff
@@ -139,6 +139,7 @@ The following tables lists the configurable parameters of the rook-operator char
 | `agent.nodeAffinity`               | The node labels for affinity of `rook-agent` (***)                                                                             | <none>                                                    |
 | `admissionController.tolerations`  | Array of tolerations in YAML format which will be added to admission controller deployment.                                    | <none>                                                    |
 | `admissionController.nodeAffinity` | The node labels for affinity of the admission controller deployment (***)                                                      | <none>                                                    |
+| `allowMultipleFilesystems`         | Allows multiple filesystems to be deployed to a ceph cluster (experimental)                                                    | `false`                                                   |
```

Comment:
Nit: I would suggest putting "experimental" before the description to make sure readers are less likely to overlook that detail.